Altis_Life.Altis/stringtable.xml

```diff
@@ -3315,7 +3315,7 @@
             <Spanish>Fallastes la misión, porque moristes.</Spanish>
             <Russian></Russian>
             <German>Da du gestorben bist, hast du es nicht geschafft das Paket abzuliefern.</German>
-            <French>Vous êtes décédé, livraison annulée.</French>
+            <French>Vous êtes mort, livraison annulée.</French>
```

Comment:
Same meaning, "décédé" is better than only "mort".

Reply:
In games the word "decédé" is very rarely used.

Reply:
That doesn't forbid us to not use it.

Reply:
Yes but it is more common and short to use that, the rest is already marked "mort".

Reply:
+1 for "mort" (dead).
Better, shorter, most common than "décédé" (deceased)